js/app.js

```diff
@@ -0,0 +1,44 @@
+ const a = 4;
+
+ const b = 53;
+ 
+  const c = 57;
+  
+  const d = 16;
+
+  const e = 'Kevin';
+
+  a < b;
+c > d;
+'Name' _=_ 'Name';
+a + b = c;
+a + a >d;
+e = 'Kevin';
+48 != '48';
+
+
+
+
+
+
+while (true) {
+	console.log('Do not run this loop');
+}
+// it is an infinite loop bacause the last exxpression is false so it will execute the code
+const runProgram = true;
+
+while (runProgram) {
+	console.log('Do not run this loop');
+	runProgram = false;
+}
+// this is not an infinite loop because we switch the value
```

Comment:
yes.  and/but actually what happens is that trying to change the value of a `const` leads to an error